### ISSUE TITLE
[Feature Request] - Allow specific transports to ignore specific levels.

### ISSUE THREAD

**Issue by Myssiry** [ihack2712/powerlog#1]
**Example**: I do not want to log `info` level to the `DiscordWebhookTransport` so I would add it to some ignore option.

Would love to have this possible.  :heart:

**Comment by ihack2712** [comment]
Just remove it from the enum :)

**Comment by Myssiry** [comment]
Let me rephrase this.
If I have code like this.
**What I am expecting to happen**:
`logger.info()` will only log to stdOut

**What really happens**:
`logger.info()` logs to webhook and to stdOut

```ts
import { DiscordWebhookTransport, Powerlog, StdTransport } from "../../deps.ts";

export enum MyLogLevels {
    debug,
    info,
    success,
    alert,
    warn,
    critical
}

export enum webhookLogLevels {
    debug,
    success,
    alert,
    warn,
    critical
}

export const logger = Powerlog.get({
    levels: MyLogLevels,
    name: "logger"
});

const myStdoutTransport = new StdTransport({
    levels: MyLogLevels
});

const myDiscordWebhookTransport = new DiscordWebhookTransport({
    levels: webhookLogLevels,
    url: "censored url",
    formatter: data => ({
        content: null,
        embeds: [
            {
                title: webhookLogLevels[data.level],
                description: data.message,
                timestamp: data.timestamp.toJSON()
            }
        ]
    })
});

await logger.use(myStdoutTransport, myDiscordWebhookTransport);
```

**Comment by ihack2712** [comment]
Yes, the example you showed will crash, all transports must have the same LogLevels. I suggest you remove the webhookLogLevels enum. On all transports you should have `enable` and `disable` methods, which are used for enabling and disabling specific log levels, so if you want to disable  the info log level you simply do the following:

```ts
const myDiscordWebhookTransport = new DiscordWebhookTransport(...)
    .disable(MyLogLevels.info);
```

That will force only the DiscordWebhookTransport to ignore log entries that uses the `info` log level. Powerlog is dynamic in the way it behaves around log levels, you can specify any log level you'd like in the `MyLogLevels` enum. What's important though is that all transports share the same LogLevel enumerable as the Powerlog instance.

Hope that helps! :)

**Comment by Myssiry** [comment]
This is exactly what I wanted.
Did not know about `.disable()`.
Thank you very much

**Comment by ihack2712** [comment]
Glad you got it working! :D
